openedx/core/djangoapps/content/learning_sequences/migrations/0005_coursecontext_days_early_for_beta.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.15 on 2020-09-01 07:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('learning_sequences', '0004_coursecontext_self_paced'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='coursecontext',
+            name='days_early_for_beta',
+            field=models.FloatField(blank=True, null=True),
```

Comment:
This still has the FloatField instead of an integer. Do you need to rebuild the migration?